mempool/src/verify.rs

```diff
@@ -241,7 +241,7 @@ pub(crate) async fn verify_tx<'a>(
         // If the recipient is not already in the mempool, then we need to check if the transaction
         // can succeed.
         if !StakingContract::can_create(accounts_tree, &db_txn, data) {
-            log::debug!("Outgoing staking transaction cannot pay fee.");
+            log::debug!("Outgoing staking transaction cannot succeed");
             return Err(VerifyErr::NotEnoughFunds);
```

Comment:
Mmm, the error indicates that this will happen because we don't have enough funds. Wondering if we should have a different error or if the `can_create` function would fail only because we can't pay the fee